database/migrations/default/1739567740039_afd_trigger_updates/up.sql

```diff
@@ -0,0 +1,53 @@
+-- add unique constraint to incident number
+-- make calendar_year a numeric column
+-- make flagged_incs a numeric column
+alter table afd__incidents
+add constraint unique_incident_number unique (incident_number),
+alter column calendar_year type numeric using (calendar_year::numeric),
+alter column flagged_incs type numeric using (flagged_incs::numeric);
```

Comment:
same as the EMS table, we are adding a missing unique constraint. i also changed some data types to `numeric` since their values are numbers.